src/breadcrumb/breadcrumb.component.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { Component } from "@angular/core";
+import { CUSTOM_ELEMENTS_SCHEMA } from "@angular/core";
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { FormsModule } from "@angular/forms";
+import { By } from "@angular/platform-browser";
+import { BreadcrumbComponent } from "./breadcrumb.component";
+import { BreadcrumbItemComponent } from "./breadcrumb-item.component";
+import { OverflowMenu } from "../dialog/overflow-menu/overflow-menu.component";
+
+@Component({
+	selector: "test-nooverflow-cmp",
+	template: `
+	<ibm-breadcrumb noTrailingSlash={{true}} overflowThreshold={{4}}>
+		<ibm-breadcrumb-item href="#">
+			Breadcrumb 1
+		</ibm-breadcrumb-item>
+		<ibm-breadcrumb-item href="#">
+			Breadcrumb 2
+		</ibm-breadcrumb-item>
+		<ibm-breadcrumb-item href="#">
+			Breadcrumb 3
+		</ibm-breadcrumb-item>
+		<ibm-breadcrumb-item href="#">
+			Breadcrumb 4
+		</ibm-breadcrumb-item>
+	</ibm-breadcrumb>`,
+	entryComponents: [BreadcrumbComponent]
+})
+class NoOverflowComponent { }
+
+@Component({
+	selector: "test-overflow-cmp",
+	template: `
+	<ibm-breadcrumb noTrailingSlash={{true}} overflowThreshold={{4}}>
```

Comment:
these could be passed in as `[noTrailingSlash]="true"` and `[overflowThreshold]="4"`